diff --git a/ajalibraries/ajantv2/includes/ntv2devicefeatures.h b/ajalibraries/ajantv2/includes/ntv2devicefeatures.h
index 4dc2bd5..cd1972e 100644
--- a/ajalibraries/ajantv2/includes/ntv2devicefeatures.h
+++ b/ajalibraries/ajantv2/includes/ntv2devicefeatures.h
@@ -10,7 +10,7 @@
 #define NTV2DEVICEFEATURES_H
 
 #if defined(AJALinux) || defined(AJA_LINUX)
-	#include <stddef.h>		// For size_t
+	#include <linux/stddef.h>		// For size_t
 #endif
 
 #include "ajaexport.h"
